File: src/main/java/ar/edu/um/tesoreria/rest/model/Curso.java
/**
 * 
 */
package ar.edu.um.tesoreria.rest.model;

import java.io.Serializable;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import jakarta.persistence.Table;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.NoArgsConstructor;

/**
 * @author daniel
 *
 */
@Data
@Entity
@Table
@EqualsAndHashCode(callSuper=false)
@NoArgsConstructor
@AllArgsConstructor
public class Curso extends Auditable implements Serializable {
	/**
	 * 
	 */
	private static final long serialVersionUID = -6015043428683272219L;
	
	@Id
	@Column(name = "cur_id")
	private Integer cursoId;
	
	@Column(name = "cur_nombre")
	private String nombre;
	
	private Integer claseChequeraId;

}

File: src/main/java/um/tesoreria/rest/kotlin/model/Curso.kt
package um.tesoreria.rest.kotlin.model

import um.tesoreria.rest.model.Auditable
import jakarta.persistence.Column
import jakarta.persistence.Entity
import jakarta.persistence.Id
import jakarta.persistence.Table

@Entity
@Table
data class Curso(

    @Id
    @Column(name = "cur_id")
    var cursoId: Int? = null,

    @Column(name = "cur_nombre")
    var nombre: String? = null,

    var claseChequeraId: Int? = null,

): Auditable()
